xml/admin-keystone.xml

```diff
@@ -83729,9 +83729,6 @@ class Something(object):
             </varlistentry>
           </variablelist>
         </section>
-        <section xml:id="module-keystone_module-contents">
-          <title>Module contents</title>
```

Comment:
This one is particular weird. These sections should belong to the API content. I'll investigate further, this one is weird.

Reply:
So is it weird or is it weird weird?

Reply:
Weird weird. It's all API config content. How did that get converted with the admin guide? Removed here:  https://github.com/SUSE-Cloud/doc-cloud/pull/800/commits/4f79d3f128c377fe3429c2ea12c961b656ae1124

Reply:
Basically, everything from here: https://docs.openstack.org/keystone/pike/configuration/index.html
Got converted too.